R/methods-Vision.R

```diff
@@ -115,6 +117,11 @@ setMethod("Vision", signature(data = "matrixORSparse"),
                     )
                 data <- data[ !(rownames(data) %in% toRemove), , drop = FALSE]
             }
+
+            if (!is.null(tree)) {
+                data = data[,tree$tip.label]
```

Comment:
I think you'll first want to take the intersection between tree & expr matrix and subset both down to the intersection. We're not always guaranteed to have every tip in the tree appear in the expression matrix.

Reply:
I think I included this in the vignettes already (I also think it works if the tree has extra anyways, just not the matrix).

Reply:
I think it's more elegant to do this within the VISION constructor, not expect users to do this.

I'm not sure that this will work if the tree has more cells -- because what if a NN does not appear in the expression matrix? Either way, it creates extra work that you don't need to do.

Reply:
Modified!